controller/src/main/java/org/jboss/as/controller/ProxyOperationAddressTranslator.java

```diff
@@ -48,6 +55,31 @@ public PathAddress translateAddress(PathAddress address) {
                 return translated;
             }
 
+            public PathAddress restoreAddress(PathAddress opAddress, PathAddress translatedAddress) {
+                // original is /host=foo/server=bar/subsystem=*
+                // translated is /subsystem=*
+                // we want the result to be translated back to /host=foo/server=bar/subsystem=quux
```

Comment:
This comment stumped me a bit. In my head an address with wildcards should expand into a list. After reading the Jira it makes sense though.